demo-certs/certs/trust-anchor.properties

```diff
@@ -0,0 +1,11 @@
+#******************************************************************************
+#**
+#** The values in this file are generated by the script 'demo-certs/create_certs.sh' when a new CA cert is created. 
+#**
+#******************************************************************************
+#
+# The CN of the certificate used as trust anchor of a tenant for client certificate based authentication.
+trusted-ca.subject-dn=CN=ca,OU=Hono,O=Eclipse IoT,L=Ottawa,C=CA
+#
+# The public key of the CA in the format that the tenant API expects for setting up a trust anchor.
+trusted-ca.public-key=MIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEApK5L6yUknQnj4FREQqs/G+klIhEKc1X4q4KrSVvFa7Vc4NW2ITTZ9X89i1AGz3gqUUZNOpN1EMNMeJkqewoYI5Bqbr4hiXC6gfUHpgiNxOBq84u/xgqNvOqrQyrz2MMivLDRfwPLbkS+/dXR33Ebh0iFJFHtceH8Vi7Fq7QIKG90+bNkzf+u3r57uBOvEjUZ96tMXaadDGmpjcaNEHp2XSZRQ/vdhSqMYKS28IRdDYuBTXguSxA7MTAtGhaqaIfyqLLufzoqc8FEyNnHKK1QQLGKIv0kvvN1OZUaaA2M0pjZh+ELs3omtBTaw03TfuqAF4bUwtv5MnPXrc25hEU1UhcCc1YOoB7KMZVUP7uP1Ua4nDBChgdRw3hDhQP+IPe5JL92780tFoQ+aFHMtp7ydyhGJGHo00mW7+5h8cGFvAkQbGf+i/tmA5zCdhBsH1f+BOknVMsQiyKb+FQk725qhe1HLZQfiU4vAjmwZLWKPESs0AquGGdFcjrjmdt2WUQ0l2dRaJLNOLxYHaxEYjvko4KPdPLpJIoKKLy/8uLrES4ervczJUlHRhbGDaWMD4NhBueyfrM1NNGtnexLTEzn8SEDwyoP9pv2OzkvdZiFKUcAl6HSV+Davh8lyb0fjQNCDKG2d+kLyxzUyhuChXTCt46seF3JjZwVmHBbHbQrBkkCAwEAAQ==
```

Comment:
Is this value consistent with the current content of the ca-cert.pem file?
If the key pair that you have created for the device has been created using the script, then the other keys and certs have been re-created as well and would need to be included in this PR ...

Reply:
Yes, it is. I took care that the current CA cert is used and that I did not commit any changed cert or key files to keep this PR as small as possible
But: I found that the CA cert expires in a week. So I hoped to get this PR merged soon and then I would provide another one with new certificates.

Reply:
Given that the certs expire anyway, FMPOV it would make sense to re-generate all keys and certs using the script and include everything in this PR. Then we can also be sure that everything is in a consistent state ...

Reply:
Oh, doing that I found that the script does not create a cert for the AMQP adapter. I will fix this as well.

Reply:
The cert for Hono Messaging can be removed?

Reply:
Done.